### PR TITLE
Make PARTS and TargetTypeValue rules more explicit

### PR DESCRIPTION
**Pull request by robUx4**
And add a basic audio example.

### CODE REVIEW

**Comment by robUx4** [comment]
The replacement rule is a bit tricky as for the TITLE the album Title should still be kept as such. Maybe we should not use any replacement and keep each element at its level.

File: matroska_tags.xml [hunk 1]
```diff
@@ -11,7 +11,9 @@
     </class>
     <class name="Entities"/>
     <class name="Search and Classification"/>
-    <class name="Temporal Information"/>
+    <class name="Temporal Information">
+      <description lang="en">All tags in this section use the Date format defined in (#date-tags-formatting)..</description>
```
Comment (retokromer) [inline matroska_tags.xml:15]: Are the two periods at the end correct?

Comment (robUx4) [reply matroska_tags.xml:15]: No..

Comment (robUx4) [reply matroska_tags.xml]: Fixed. Thanks for your attention!

**Pull request merged**